Dockerfile

```diff
@@ -5,6 +5,7 @@ ENV LANG=C.UTF-8
 
 # System packages
 RUN apt-get update && \
+  apt-get upgrade -y && \
```

Comment:
@whereismyjetpack Is this a typical thing to do? It seems you're introducing uncertainty here because each time you deploy, you're changing the make up of the underlying system. How do you ensure that you're getting the same system each time, or is that not a concern.

Reply:
This is always a tricky balance. Since we are building a base image based off the Dockerfile and Gemfiles the underlying system won't change between "stages" and any system update that breaks things should be found early. I think this is a fair balance between stability, and patching vulnerabilities.